frontend/src/App.js

```diff
@@ -1,13 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Footer from './components/Partials/Footer/Footer';
 import Header from './components/Partials/Header/Header';
 
 function App(props) {
+  let scrollTop;
+
+  useEffect(() => {
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    scrollTop = document.querySelector('#scroll-top');
+    window.addEventListener('scroll', scrollHandler, false);
+
+    return () => {
+      window.removeEventListener('scroll', scrollHandler);
+    };
+  }, []);
+
+  function toScrollTop() {
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth',
+    });
+  }
```

Comment:
```suggestion
  function scrollToTop() {
    window.scrollTo({
      top: 0,
      behavior: 'smooth',
    });
  }
```
Ngoài ra có thể tạo 1 custom hook cho phần scroll này, vì có thể nó sẽ được dùng khá là nhiều
Vd như khi change pagination ở product list sử dụng cũng được

Reply:
Không cần đâu anh ạ, vì em viết scrollToTop() ở App.js là tất cả các trang sau đã có rồi.  Vì trong phần folder router em đã cấu hình ngon rồi!